chips/nrf51/src/radio.rs

```diff
@@ -34,34 +35,33 @@ impl Radio {
     pub const fn new() -> Radio {
         Radio {
             regs: peripheral_registers::RADIO_BASE as *const peripheral_registers::RADIO_REGS,
-            txpower: Cell::new(0),
+            tx_power: Cell::new(TxPower::ZerodBm),
             rx_client: Cell::new(None),
             tx_client: Cell::new(None),
         }
     }
 
 
-    fn ble_init(&self, channel: RadioChannel) {
+    fn ble_init(&self, channel: RadioFrequency) {
```

Comment:
I changed because it is RF channels in the Bluetooth Standard and the variants of the enum are named `DataChannelX` / `AdvertisingChannelY`.

But, I agree RadioChannel is better especially when reading only the type name as in this case!
I will change it back